components/topbar/user/src/index.scss

```diff
@@ -147,6 +161,21 @@
             $background-color: $bgc-topbar-user-cta,
             $icon-fill: $c-white
           );
+          position: relative;
+        }
+
+        &Notifications {
+          background-color: $c-primary;
```

Comment:
and done!

I don't know if what i did is what you wanted, or just keep top & right outside the mixin?